File: StripeExpressPayoutsDemo/app/src/main/java/com/example/stripe_express_payouts_demo/MainActivity.java
package com.example.stripe_express_payouts_demo;

import androidx.appcompat.app.AppCompatActivity;

import android.accounts.Account;
import android.os.Bundle;

import com.stripe.android.Stripe;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    private void initStripe(){

    }
}

File: AndroidDemo/app/src/main/java/com/example/androiddemo/MainActivity.kt
package com.example.androiddemo

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.util.Log
import androidx.activity.viewModels
import androidx.lifecycle.lifecycleScope
import dagger.hilt.android.AndroidEntryPoint
import kotlinx.coroutines.flow.collectLatest
import kotlinx.coroutines.launch

@AndroidEntryPoint
class MainActivity : AppCompatActivity() {

    private val viewModel:MainViewModel by viewModels()

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        getMovieData()
    }

    private fun getMovieData(){
        lifecycleScope.launch {
            viewModel.getMovieList("marvel").collectLatest { result ->
                when(result){

                    is NetworkResult.Loading -> {
                        Log.d("Krish","Loading")
                    }

                    is NetworkResult.Success -> {
                        Log.d("Krish",result.data.Search.toString())
                    }

                    else -> {}
                }
            }
        }
    }








}